src/common/addlayers/RegistryLayersDirective.js

```diff
@@ -269,6 +291,15 @@
 
             // load saved registry layers if they exist
             scope.addRegistryLayersFromSavedMap();
+
+            function addCartLayersInMapPreview(layerConfig) {
+              layerConfig.CRS = ['EPSG:4326'];
```

Comment:
The map preview is coming from mapproxy and it is safe to assume it is 4326. Bounding boxes are normalized to EPSG:4326 before sending to elasticsearch.

The original SRS is still kept in the metadata and mapproxy is able to reproject if needed.